server/middleware/auth.ts

```diff
@@ -1,31 +1,48 @@
 import jwt from "jsonwebtoken";
-import type { Request, Response, NextFunction } from 'express'
+import type { Request, Response, NextFunction } from "express";
 
-export const auth = async (req: Request,   res: Response, next: NextFunction) => {
+import type { UserRole } from "../types/api";
+
+export const auth = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+  roles?: UserRole[]
+) => {
   try {
-    const accessToken = req.headers.authorization?.split(" ")[0];
+    const accessToken = req.headers.authorization?.split(" ")[1];
```

Comment:
Is this works in every system .
We need to maintain proper way.
For postman changes don't push it.

Reply:
Yup, It works fine in all the systems.
I have committed it previously since I was not knowing how we need to send tokens so.